lib/config/config-validator.js

```diff
@@ -181,10 +186,10 @@ function formatErrors(errors) {
  * @returns {void}
  */
 function validateConfigSchema(config, source) {
-    const validator = schemaValidator(configSchema, { verbose: true });
+    validateSchema = validateSchema || ajv.compile(configSchema);
```

Comment:
Personally I'd put this `function` to block code with `validateSchema` variable like:
```js
{
    let validateSchema;

    /**
     * ...
     */
    function validateConfigSchema(...) {
        validateSchema = validateSchema || ajv.compile(configSchema);
        // ...
    }
}
```
It would to read this part of code as well as help v8 to reduce variables in context.

Or at least just put it closer to this function.